packages/esm-patient-vitals-app/src/vitals/vitals.resource.tsx

```diff
@@ -244,15 +244,15 @@ export function editPatientVitals(
 }
 
 export function assessValue(value: number, range: ObsMetaInfo): ObservationInterpretation {
-  if (range?.hiCritical && value >= range.hiCritical) {
+  if ((range?.hiCritical && value >= range.hiCritical) || (range?.hiAbsolute && value >= range.hiAbsolute)) {
```

Comment:
We should presumably also apply whatever logic we have for absolute values to blood pressures (though I realise that in CIEL, these are undefined).

Reply:
Maybe we can just flag the inputs that are not within the absolute range using carbon invalid input styles like it is now on dev3, however, an input like the temperature has a range of `23 - 43`, so I would assume 39.5 degrees Celsius should show as critical but the temperature concept doesn't return `hiCritical`, so that means they won't be flagged yet some could need clinical attention.

Reply:
> using carbon invalid input styles 

I think that's preferable. We want to be able to distinguish "clinically concerning" from "have you checked to make sure that's a patient and not just a ball of fire"?

> but the temperature concept doesn't return

Let's solve this on the concept dictionary / reference range side of things. I think hard-coding validation logic about clinical significance should be avoided as much as possible—we want the system to be flexible enough to meet implementer needs.

Reply:
sure

Reply:
<img width="431" alt="Screenshot 2023-10-25 at 00 08 13" src="https://github.com/openmrs/openmrs-esm-patient-chart/assets/30952856/612bc117-6ea8-46ec-855d-cb524fb3b67c">

@ibacher what do you think about this?

Reply:
Maybe I should make the border cover the who container 🤔 . cc @denniskigen

Reply:
See the screenshot I've posted in my most recent comment below.